run_integration_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+. ./src/lib/kw_include.sh --source-only
+include './integration_tests/utils.sh'
```

Comment:
You should not use test features in this script. Here, you should only care about:
1. Setup
2. Run specific test
3. Run all tests

You should have something like:
```sh
function prepare()
{
# Check if the container is ready for use
# If not, initialize the container
}

function run_single_test()
{
   local test_name="$1"
  # Check if test_name exist in the integration_test/ folder
}

function run_all_tests()
{
 # Check all tests available in the integration_test folder, and run all of them.
}
...
```